src/ToolBox/SOS/Strike/strike.h

```diff
@@ -76,6 +76,21 @@
 #ifndef PAL_STDCPP_COMPAT
 #include <malloc.h>
 #endif
+
+#if defined(_MSC_VER)
+#define alloca _alloca
```

Comment:
I think we still need `_MSC_VER` condition in addition to what @jkotas propose in both places.